lib/helpers.js

```diff
@@ -131,3 +132,77 @@ exports.getBaseModelForDataSourceName = function(name, allDataSources) {
   return connectorMeta && connectorMeta.baseModel;
 };
 
+/**
+ * Get the available type choices that should be used for arguments
+ * from workspace.
+ */
+exports.getTypeChoices = function() {
+  var typeChoices = ModelProperty.availableTypes.concat({
+    name: '(other)',
+    value: null
+  });
+  return typeChoices;
+};
+
+/**
+ * Print a sample dummy method implemented in `modelName.js` file
+ * @param {String} modelName Name of model
+ * @param {String} methodName Name of remote method
+ * @param {Boolean} isStatic Method defined on constructor with value true
+ *  while defined on prototype with value false
+ * @param {Array} accepts An array of accept arguments
+ * @param {Array} returns An array of return arguments
+ */
+exports.printDummyMethod = function(
```

Comment:
@jannyHou Do you envision to call this method from other places than the remote-method generator? If not then it would be better to keep the implementation in `remote-method/index.js`.